Test/dafny0/snapshots/Snapshots0.run.dfy.expect

```diff
@@ -2,7 +2,7 @@ Processing command (at Snapshots0.v0.dfy(4,10)) assert Lit(false);
   >>> DoNothingToAssert
 
 Dafny program verifier finished with 1 verified, 0 errors
-Processing call to procedure Call$$_module.__default.bar in implementation Impl$$_module.__default.foo (at Snapshots0.v1.dfy(3,6)):
+Processing call to procedure bar (call) in implementation foo (correctness) (at Snapshots0.v1.dfy(3,6)):
```

Comment:
I find the parentheses a bit confusing in this example.  Might be a problem for later, though.

Reply:
No, the module path is included when explicit. (See the changes in `Superposition.dfy.expect`, for example.)